File: frontend/src/dto/InfoDTO.ts
export interface InfoDTO {
    spotId: number;
    spotName: string;
    windows: WindWindow[];
    spotLatitude: number,
    spotLongitude: number
}

export interface WindWindow {
    windWindowId: number;
    speed: number;
    startAngle: number;
    endAngle: number;
}

File: backend/src/main/java/de/windalert/dto/InfoDTO.java
package de.windalert.dto;

import java.util.List;

public record InfoDTO(
        Long spotId,
        String spotName,
        Float spotLatitude,
        Float spotLongitude,
        List<WindWindow> windows
) {
    public record WindWindow(
            Long windWindowId,
            Integer speed,
            Integer startAngle,
            Integer endAngle
    ) {
    }
}
